File: DSA/VikrantSinghBanwala_590018082/day2question2.c
#include<stdio.h>
void fandl(int nums[],int n){
    if (n>0){
        printf("First:%d,Last:%d\n",nums[0],nums[n-1]);
    }
}
int main(){
    int nums[]={1,2,3,4};
    int n=sizeof(nums)/sizeof(nums[0]);
    fandl(nums,n);
    return 0;
}

File: DSA/LakshaKundra_590013237/Day_2_Question_2.py
def first_and_last(nums):
    first = nums[0]
    last = nums[-1]
    return first, last
nums1 = [1, 2, 3, 4]
nums2 = [7]
nums3 = [5, 9, 2]
print("First:", first_and_last(nums1)[0], "Last:", first_and_last(nums1)[1])
print("First:", first_and_last(nums2)[0], "Last:", first_and_last(nums2)[1])
print("First:", first_and_last(nums3)[0], "Last:", first_and_last(nums3)[1])
